cypress/e2e/HW/HW19.1.cy.js

```diff
@@ -0,0 +1,37 @@
+import garagePageSteps from "../../support/steps/GaragePage"
+
+const login = Cypress.env('login')
+const password = Cypress.env('password')
+
+ 
+// describe ('Log into Hillel', ()=> {
+    
+//     it ('User can log into account with correct credentials', ()=> {
+//         cy.visit('/')
+//         cy.get('.btn.btn-outline-white.header_signin').click()
+//         cy.get('#signinEmail').type(login)
+//         cy.get('#signinPassword').type(password, { sensitive: true })
+//         cy.get('.form-check-label').click()
+//         cy.get('.btn.btn-primary').contains('Login').click()
+//       })
+// })
+
+describe ('Verifying garage page', ()=> {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.get('.btn.btn-outline-white.header_signin').click()
+        cy.get('#signinEmail').type(login)
+        cy.get('#signinPassword').type(password, { sensitive: true })
+        cy.get('.btn.btn-primary').contains('Login').click()
+        })
+
+    it('user can add a car into a garage', ()=> {
+        garagePageSteps.addCar()
+    })
+
+    it('user can add fuel expense to the car', ()=> {
+        garagePageSteps.addCar()
+        garagePageSteps.addFuelExpense()
```

Comment:
pass arguments into methods. That way you will be able to reuse them with any proper data

Reply:
i see)
thanks